docs/src/main/sphinx/release/release-456.md

```diff
@@ -0,0 +1,64 @@
+# Release 456 (4 Sep 2024)
+
+## General
+
+* Expose additional JMX metrics about resource groups, including CPU and memory
+  usage, limits, and scheduling policy. ({issue}`22957`)
+* Add [adaptive plan optimization](/optimizer/adaptive-plan-optimizations) which
+  can reorder joins at runtime. The feature can be disabled with the
+  `fault-tolerant-execution-adaptive-join-reordering-enabled` configuration
+  property or the `fault_tolerant_execution_adaptive_join_reordering_enabled`
+  session property. ({issue}`23046`)
+
+## JDBC driver
+
+* Add publishing a JDBC driver JAR without shaded, third-party dependencies. ({issue}`22098`)
+
+## BigQuery connector
+
+* Fix failures with queries using table functions when `parent-project-id` is
```

Comment:
@pajaks and @ebyhr is this okay? I dont think we can use PTF in the release notes and I am not sure if this is now correct. Did you mean only the query pass through table function?

Reply:
Looks good to me